aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. 
+
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
```

Comment:
```suggestion
* Compatibility aspects.
```

Reply:
Let's not minimize limitations by leaving it out. Also my intro was removed but it needs some intro, so I added the shorted possible.

```suggestion
ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. This article shows:

* How to get started.
* The benefits and potential drawbacks.
* The limitations and compatibility.
```